internal/helper/yawollet.go

```diff
@@ -301,6 +301,7 @@ func createEnvoyTCPListener(
 	listenPort, err := anypb.New(&envoytcp.TcpProxy{
 		StatPrefix:       "envoytcp",
 		ClusterSpecifier: &envoytcp.TcpProxy_Cluster{Cluster: fmt.Sprintf("%v-%v", port.Protocol, port.Port)},
+		IdleTimeout:      &duration.Duration{Seconds: 300}, // default is one hour, prevent idle connection in cases of openstack server migration
```

Comment:
```suggestion
		IdleTimeout:      &duration.Duration{Seconds: 60}, // default is one hour, prevent idle connection in cases of openstack server migration
```

can be even lower. AWS uses 60s.

Reply:
I researched more and see that this is a complicated topic. Some applications can have problems with low settings. I decided to make it configurable and leave the default as it is, to not break something.